.github/CONTRIBUTING.md

```diff
@@ -3,6 +3,18 @@
 Please take a moment to review this document in order to make the
 contribution process easy and effective for everyone involved.
 
+## Contributing code
```

Comment:
There's also the existing "Contributing Code" section further down. We should probably merge the two or give them different titles.

Reply:
I've merged both sections.